elibrary/templates/elibrary/welcome.html

```diff
@@ -1,2 +1,67 @@
+{% extends "elibrary/base.html" %}
+{% block content %}
+{%load static%}
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="{% static 'elibrary/welcome.css' %}">
+  </head>
+  <body>
 
+   <div class="sub-box-0" id="sub-box-0">
+    <h1 class="boxheading">WELCOME TO THE E-LIBRARY</h1>
+    <h2 class="boxheading2">Available books</h2>
+    <h3 class="boxheading3"><li>principles of computer science</li>
+        <li>OOP</li>
+        <li>software engineering</li>
+        <li>data analytics</li>
+        <li>machine learning</li>
+        <li>artificial intelligence</li>
+        <li>data structures</li>
+   </div>
+
+  </body>
+
+  <footer>
+    <div class="sub-box-1" id="sub-box-1">
+      <!-- Display the countdown timer in an element -->
+      <div class="teext">TECHNICAL BOOKS SALE STARTS IN</div>
+      <div class="timer" id="demo"></div>
+      <script>
+        // Set the date we're counting down to
```

Comment:
It's a good thing that you have already added comments but it would be more readable to leave some space between lines.